slave/slave_files/files/home/jenkins-slave/cleanup_docker_images.py

```diff
@@ -54,14 +54,16 @@ def check_done(args):
 
 
 def print_progress(args):
-    print("Free space %.2fGb of %s required" % (get_free_disk_space(args.path),
-                                                args.minimum_free_space))
-    print("Free space %.2f%% of %s required" % (get_free_disk_percentage(args.path),
-                                                args.minimum_free_percent))
+    logging.info("Free space %.2fGb of %s required" %
+                 (get_free_disk_space(args.path),
+                  args.minimum_free_space))
+    logging.info("Free space %.2f%% of %s required" %
+                 (get_free_disk_percentage(args.path),
+                  args.minimum_free_percent))
 
 
 def run_cleanup(args):
```

Comment:
Since there is now also a `run_container_cleanup` this should be renamed to `run_image_cleanup`.